include/blmc_robots/n_joint_blmc_robot_driver.hpp

```diff
@@ -734,3 +348,5 @@ typename Driver::Types::BackendPtr create_backend(
 }
 
 }  // namespace blmc_robots
+
+#include "n_joint_blmc_robot_driver.hxx"
```

Comment:
optional : include can be inside the namespace, which scope does not need to be reopened in the hxx file (not sure if one has any advantage of the other)

Reply:
I prefer it like this as it makes the hxx file more self-contained (and also more consistent with how cpp files look).